jsons/Policies.json

```diff
@@ -501,7 +501,7 @@
 			{
 				"name": "Skyscrapers",
 				"uniqueText": "Cost of purchasing buildings reduced by [50]%",
-				"uniques": ["Cost of purchasing [Science] buildings reduced by [50]%","Cost of purchasing [Production] buildings reduced by [50]%","Cost of purchasing [Food] buildings reduced by [50]%","Cost of purchasing [Gold] buildings reduced by [50]%","Cost of purchasing [Happiness] buildings reduced by [50]%","Cost of purchasing [Culture] buildings reduced by [50]%"],
+				"uniques": ["[-50]% Gold Cost of purchasing [Non-Military] Buildings"],
```

Comment:
Again, "Non-Military" isn't a valid buildingFilter, so you'd need to use the old filters or do something else